app/models/community.rb

```diff
@@ -34,4 +36,19 @@ def slug_candidates
         [:name, :category,:url]
       ]
     end
+
+    def accept_proflie_image
+      return unless profile_image.attached?
+      profile_acceptable_types = ["image/jpeg", "image/png", "image/gif"]
+      unless profile_acceptable_types.include?(profile_image.content_type)
```

Comment:
since this is repeated in multiple places, can we create a common helper method? and avoid duplicates everywhere?

Reply:
Resolved  it